source/stylesheets/home.sass

```diff
@@ -69,4 +69,14 @@ section.content .products-page
     padding: var(--med-padding)
 
   @media screen and (max-width: $break-small)
-    padding: var(--small-padding)
\ No newline at end of file
+    padding: var(--small-padding)
+
+a.all-products-button
+  max-width: 350px
+  background: none
+  color: var(--primary-text-color)
+  border: 2px solid var(--primary-text-color)
+  gap: 12px
+
+  svg
+    fill: var(--primary-text-color)
```

Comment:
this should be `fill: currentColor` so it plays nice with hover states

Reply:
done!